apps/portal/src/content/docs/components/form.mdx

```diff
@@ -7,63 +7,144 @@ The `Form` component is a container for form elements, providing a structured la
 
 import { DocsPage } from "@/components/docs-page";
 
+<DocsPage.ComponentExampleWithZodAndForm
+  client:only
+  code={`() => {
```

Comment:
The formatting of this code block is a bit all over the place. Unfortunately prettier only sees it as a big string and not code, so you'll have to do it manually; or copy it to a new doc, run formatting and then paste it back in.

Reply:
By doing so, prettier pipeline is failing. reverting it.